lib/runners/php.js

```diff
@@ -18,13 +18,29 @@ module.exports.run = function run(opts, cb)
         },
         fullProject: function ()
         {
-            throw 'Test framework is not supported'
-        }
-    });
-};
-
-
-
+            var code = [
+                config.snippets.php.bootstrap,
+                (opts.setup || ''),
+                config.snippets.php.beforeUserScript,
+                opts.solution,
+                config.snippets.php.afterUserScript,
+                config.snippets.php.classBegin,
+                opts.fixture,
+                config.snippets.php.classEnd,
+                config.snippets.php.execute
+            ].join('\n');
 
+            return {name: 'php', 'args': ['-r', code]};
+        },
+        sanitizeStdOut : function (output) {
+            var lines = output.split('\n');
+            var success = (lines[lines.length - 2] || '').slice(0, 2) === 'OK';
```

Comment:
Hi Codewars/codewars-runner,
A message from Matthew Wampler-Doty was left on IQ Elite for you. 
Click here a.iqelite.com/j/0JmRlo/f3r/?hg=J4GBUSzi6k to read your message 


The IQ Elite Team



---------------------------------------------------------------------------
This message was meant for reply@reply.github.com. If you do not want to continue receiving such mails from IQ Elite, please click on this link: a.iqelite.com/common/unsubscribe/?code=00d15143&email=reply@reply.github.com&hg=J4GBUSzi6k
IQ Elite is a service with main office: C/ Gran De Gràcia 15, 1a 1a, 08012 Barcelona, Spain.
---------------------------------------------------------------------------